src/us_ofac.rs

```diff
@@ -0,0 +1,104 @@
+use crate::{ConfigError, Location};
+use std::collections::hash_map::Entry::{Occupied, Vacant};
+use std::collections::{HashMap, HashSet};
+
+fn get_non_passing_entries() -> HashMap<String, Option<HashSet<String>>> {
+  HashMap::from([
+    (
+      String::from("UA"),
+      Some(HashSet::from([String::from("Crimea")])),
+    ),
+    (String::from("IR"), None),
+    (String::from("SY"), None),
+    (String::from("CU"), None),
+    (String::from("KP"), None),
```

Comment:
Optional/DRY: Use `into()` instead of `String::from` (also in tests below)

```suggestion
      "UA".into(),
      Some(HashSet::from(["Crimea".into()])),
    ),
    ("IR".into(), None),
    ("SY".into(), None),
    ("CU".into(), None),
    ("KP".into(), None),
```